cmd/skynet/skynetcmd.go

```diff
@@ -70,6 +70,53 @@ fetch a skylink file from a chosen skynet portal.`,
 		Run: wrap(skynetpincmd),
 	}
 
+	skynetSkykeyCmd = &cobra.Command{
+		Use:   "skykey",
+		Short: "Perform skykey operations.",
+		// A subcommand must be provided.
```

Comment:
Q: will this still result in a usage output if someone types `skynet skykey`? If so great, if not can it?

Reply:
Yep:

```
$ skynet skykey
Perform skykey operations

Usage:
  skynet skykey [command]

Available Commands:
  add         Add a skykey to Skynet.
  create      Create a skykey on Skynet.
  get         Perform skykey get operations
```

Reply:
great!